includes/class-sensei-lesson.php

```diff
@@ -268,14 +268,14 @@ public function lesson_info_meta_box_content() {
 	}
 
 	/**
-	 * lesson_prerequisite_meta_box_content function.
+	 * Lesson_prerequisite_meta_box_content function.
```

Comment:
It would be nice to replace this with a brief description of what this function actually does.

Reply:
Done!